go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/golang/glog v1.0.0
 	github.com/golang/protobuf v1.5.2
```

Comment:
github.com/golang/protobuf is deprecated and should be replaced with google.golang.org/protobuf